UnityProject/Assets/Scripts/Weapons/Gun.cs

```diff
@@ -37,7 +37,8 @@ public class Gun : NetworkBehaviour, IPredictedCheckedInteractable<AimApply>, IC
 		/// <summary>
 		///     The type of ammo this weapon will allow, this is a string and not an enum for diversity
 		/// </summary>
-		[FormerlySerializedAs("AmmoType")] public AmmoType ammoType;
+		[FormerlySerializedAs("AmmoType")] 
+		public AmmoType ammoType;
```

Comment:
Use a private property with ``SerializeField`` decorator and a public getter like this:
```suggestion
        [SerializeField]
		private AmmoType ammoType;
		
		public AmmoType AmmoType => ammoType;
```

Also please include Tooltip for all fields exposed to inspector. Using summary is alright for code you're meant to work with in your IDE, but inspector fields benefit from having tooltips so you just hover and see what the thing is about.